README.md

```diff
@@ -22,12 +22,17 @@ Develop branch status
 
 #quickstart instructions
 
+0. Clone data_api repo
+
+        git clone https://github.com/kbase/data_api/
+
 1. install virtualenv and python development libraries using your local installer::
 
         apt-get install python-dev python-virtualenv
 
 2. create a virtualenv environment to install the source to::
-
+        
+        cd data_api
         virtualenv venv
 
 3. activate the virtualenv::
```

Comment:
there is something there:

 source venv/bin/activate

you don't see that?


On Fri, Nov 6, 2015 at 11:38 AM, kkellerlbl <notifications@github.com>
wrote:

> In README.md
> <https://github.com/kbase/data_api/pull/102#discussion_r44178522>:
>
> >  1. install virtualenv and python development libraries using your local installer::
> >
> >          apt-get install python-dev python-virtualenv
> >
> >  2. create a virtualenv environment to install the source to::
> > -
> > +
> > +        cd data_api
> >          virtualenv venv
> >
> >  3. activate the virtualenv::
>
> Is there supposed to be something here?
>
> source venv/bin/activate
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/data_api/pull/102/files#r44178522>.
>